packages/components/pages/f-account-info/test/visual/f-account-info.visual.spec.js

```diff
@@ -0,0 +1,53 @@
+import forEach from 'mocha-each';
+
+const AccountInfo = require('../../test-utils/component-objects/f-account-info.component');
+
+let accountInfo;
+
+forEach(['desktop', 'mobile'])
```

Comment:
I don't think we want to put 'desktop' & 'mobile' tests in the same file, is this right @siggerzz

Reply:
actually I spoke to Ben about this and we decided we can put them in the same file - the `beforeEach` in this file ensures we get the same result, without having to duplicate the tests